sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusReactorAmqpConnection.java

```diff
@@ -197,6 +198,35 @@ public ServiceBusReactorAmqpConnection(String connectionId, ConnectionOptions co
             });
     }
 
+    @Override
+    public Mono<TransactionManager> getTransactionManager() {
```

Comment:
I'm looking through the usage of this.. and do we really need a TransactionManager class? This always calls `getSession("coordinator"), namespace, "coordinator")`, then passes any calls onto the session. Why not expose the methods as-is. (ie. `createTransaction()`) and underneath, keep that "coordinator" session. The TransactionManager class isn't managing any transactions.

Reply:
I was debating it, it provide little bit of abstraction from AmqpSession in client itself but not a big gain any way. I will remove it.